package.json

```diff
@@ -1,25 +1,28 @@
 {
-    "name": "storybook-builder-vite-parent",
-    "version": "0.1.13",
-    "description": "An experimental plugin to run and build Storybooks with Vite",
-    "main": "index.js",
-    "author": "",
-    "license": "MIT",
-    "scripts": {
-        "lint": "prettier --check .",
-        "format": "prettier --write ."
-    },
-    "workspaces": [
-        "packages/example-react",
-        "packages/example-svelte",
-        "packages/example-vue",
-        "packages/example-workspaces",
-        "packages/storybook-builder-vite"
-    ],
-    "prettier": {
-        "singleQuote": true
-    },
-    "devDependencies": {
-        "prettier": "^2.3.2"
-    }
+  "name": "storybook-builder-vite-parent",
+  "version": "0.1.13",
+  "description": "An experimental plugin to run and build Storybooks with Vite",
+  "license": "MIT",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "scripts": {
+    "lint": "eslint \"packages/*/**/*.{js,jsx,mjs,cjs}\" --fix",
```

Comment:
ESLint is not currently configured correctly for `.jsx` files, let's remove it from this list.  And actually, I think as a starting point it would be good to run eslint only on `packages/storybook-builder-vite`, and the examples should probably have their own linting configs, since react will be different from vue and from svelte, etc.

Reply:
Right now I have zero ESLint warnings and errors in #195 pull request.

So, for me It's works for all codebase (also for `*.jsx`).

If you want to provide additional configs, let's do it after this and #195 pull requests will be merged.